bin/consul-templaterb

```diff
@@ -75,6 +93,22 @@ optparse = OptionParser.new do |opts|
     options[:consul][:token] = consul_token
   end
 
+  opts.on('-V', '--vault-addr=<address>', String, 'Address of Vault, eg: http://localhost:8200') do |vault_url|
+    options[:vault][:base_url] = vault_url
+  end
+
+  opts.on('-T', '--vault-token=<token>', String, 'Token used to authenticate against vault.') do |vault_token|
+    options[:vault][:token] = vault_token
+  end
+
+  opts.on('', '--no-vault-renew', 'Disable auto-renewal of the Vault token') do
```

Comment:
Use `--vault-renew [boolean]` => and specify the default value in help

Reply:
Roger.